src/reference/environments/envs/cylc-all.yml

```diff
@@ -1,8 +1,8 @@
-name: cylc-|release|
+name: cylc-|version|
 channels:
   - conda-forge
 dependencies:
-  - cylc-flow =|release|
+  - cylc-flow =|version|
```

Comment:
Not sure why you've changes these, they are example recipes so `cylc-flow` should be pinned to the minor version, i.e. the `|release|` not the `|version|`.

Reply:
I was trying to make these consistent with what is noted here: https://github.com/cylc/cylc-doc/blob/34f8c5539652b8a9da786fa4febb5d5a7e58189f/src/conf.py#L93-L96

Reply:
That's because Sphinx defines release/version the other way around to normal hence this line:

> # NOTE: Sphinx considers version/release the other way around to us

Reply:
But the point is `version` should have been what we were using for pinning, not `release`

Reply:
Ok, tested with 8.0.1, it came out in the environment files as 8.0 so good. I find "version" meaning the short version confusing, but can deal with it.